geth/api/api.go

```diff
@@ -147,27 +147,27 @@ func (api *StatusAPI) Logout() error {
 }
 
 // SendTransaction creates a new transaction and waits until it's complete.
-func (api *StatusAPI) SendTransaction(ctx context.Context, args common.SendTxArgs) (gethcommon.Hash, error) {
+func (api *StatusAPI) SendTransaction(ctx context.Context, args utils.SendTxArgs) (gethcommon.Hash, error) {
```

Comment:
I would move it to `transactions` package to have `transactions.SendTxArgs`. I'm not sure that `SendTxArgs` is a good name either, but let's keep it for another time.

Reply:
Yep, previously I had the limitation of cyclic dependency, thats why it lives on utils. However as discussed with @adambabik I'll try to merge everything under `geth/transactions` package.